src/core_plugins/kibana/public/dashboard/dashboard_app.js

```diff
@@ -140,9 +140,6 @@ app.directive('dashboardApp', function ($injector) {
 
       timefilter.enableAutoRefreshSelector();
       timefilter.enableTimeRangeSelector();
-      dash.searchSource.highlightAll(true);
-      dash.searchSource.version(true);
```

Comment:
I'm concerned with these deletions

Reply:
its not needed for visualize and discovery sets this on its own searchsource, so we don't need it here in dashboard.